### PR TITLE
General documentation update

### PR DESCRIPTION
**Pull request by sergiusens**
Most of these changes are to reflect the new cli, but there are also
other minor corrections in here.

Signed-off-by: Sergio Schvezov <sergio.schvezov@canonical.com>

### CODE REVIEW

File: docs/snapcraft-advanced-features.md [hunk 1]
```diff
@@ -171,10 +171,9 @@ thoughts on services and their security, visit the
 
 ### Limiting the number of installed files
 
-To check the list of files included in your snap, you can use `dpkg -c` on
```
Comment (kyrofa) [inline docs/snapcraft-advanced-features.md]: Are we completely losing this functionality with the move to squashfs? Do we have anything equivalent? It was quite handy to be able to open the .snap in the Archive Manager, or use `dpkg -c`.

**Comment by kyrofa** [comment]
Overall this looks pretty good. I know we have some other documentation (the ROS one immediately comes to mind) that still says to run `snapcraft` with no arguments to create the .snap. Should the rest of the docs be updated here?

File: docs/snapcraft-advanced-features.md (hunk 1 above)
Comment (sergiusens) [inline docs/snapcraft-advanced-features.md]: found `unsquashfs -l`

**Comment by sergiusens** [comment]
@kyrofa I've updated the ros mention as well

File: docs/get-started.md [hunk 2]
```diff
@@ -10,6 +10,9 @@ technology, we recommend to use the latest classic Ubuntu Long-Term Support
 those not using an Ubuntu machine (and you should), you can use a VM
 (VirtualBox, VMware, Vagrant) to execute your Ubuntu development host.
 
+This version of snapcraft only works on xenial, for previous versions of
```
Comment (elopio) [inline docs/get-started.md]: being a pita, I would replace xenial with something like Ubuntu 16.04 (xenial)

File: docs/get-started.md [hunk 3]
```diff
@@ -10,6 +10,9 @@ technology, we recommend to use the latest classic Ubuntu Long-Term Support
...
+snapcraft, refer to the 1.x documentation.
```
Comment (elopio) [inline docs/get-started.md]: maybe a link to the 1.x branch in github?

**Comment by elopio** [comment]
lgtm. I just left a couple of ignorable comments.

**Comment by kyrofa** [comment]
Yeah, looks good to me too.

**Pull request merged**